examples/slice-pizza/datadog-app/public/_redirects

```diff
@@ -1,3 +1,3 @@
+/slice-pizza-widget    /index.html   200
```

Comment:
I'm not actually sure if this is important (and just realized that the `/slice-pizza-modal` route isn't defined here either.

Do you know if this file is actually necessary?

Reply:
Sorry, I honestly have no idea. I've been cargo-culting this stuff through the examples because I assumed it was something we wanted. I imagine it's something from `create-react-app` that helps somehow when publishing an actual site. Maybe the `create-react-app` documentation explains the purpose?